src/components/MnemonicPhrase/MnemonicPhrase.js

```diff
@@ -22,6 +22,7 @@ import styled from 'styled-components/native';
 import { BaseText, MediumText } from 'components/Typography';
 import { fontSizes } from 'utils/variables';
 import { getColorByTheme } from 'utils/themes';
+import { makeViewPrivate } from 'utils/ui';
 
```

Comment:
```suggestion
import { excludeFromMonitoring } from 'utils/appMonitoring';
```

I struggle a bit about the name suggestion. However , the important thing would be somehow connect the `makeViewPrivate` method with monitoring feature. Which currently is not obvious when `makeViewPrivate` is imported from `utils/ui`.